packages/docs/docs/rive/remotionrivecanvas.md

```diff
@@ -41,6 +45,33 @@ function App() {
 }
 ```
 
+## Set Text Run at Runtime Example
+
+This example assumes that your Rive animation has a text run named "city". See [here](https://help.rive.app/runtimes/text#low-level-api-usage) for
+more information about Text Runs on Rive.
+
+```tsx twoslash
+import { RemotionRiveCanvas } from "@remotion/rive";
+import { RiveCanvas } from "@rive-app/canvas-advanced";
+import { useCallback } from "react";
+
+// Make sure to wrap your onLoad handler on `useCallback` to avoid re-rendering this component every single time
+const onLoadHandler = useCallback((rive: RiveCanvas) => {
+  const artboard = rive.defaultArtboard();
```

Comment:
This example doesn't seem to compile in TypeScript.
`defaultArtboard` does not exist on `RiveCanvas`, but it exists on `File`. Is this what you meant?